app/src/main/java/com/babylon/wallet/android/presentation/ui/composables/Thumbnail.kt

```diff
@@ -439,6 +444,26 @@ enum class ThumbnailRequestSize(val size: Int) {
     }
 }
 
+@Suppress("MagicNumber")
+private fun ImageRequest.Builder.applyAllSupportedImageDecoders() = apply {
+    this.decoderFactory { result, options, _ ->
+        if (result.mimeType == "image/svg+xml" || DecodeUtils.isSvg(result.source.source())) {
+            SvgDecoder(result.source, options)
+        } else if (DecodeUtils.isGif(result.source.source())) {
+            GifDecoder(result.source, options)
+        } else if (Build.VERSION.SDK_INT >= 28 && (
+                DecodeUtils.isAnimatedWebP(result.source.source()) || DecodeUtils.isAnimatedHeif(
+                    result.source.source()
+                )
+                )
+        ) {
+            ImageDecoderDecoder(result.source, options)
+        } else {
+            null
```

Comment:
what is implication of executing this line? Does it impact decoding of, for example "regular" static images if we would set decoder to null?

Reply:
same question do we even need to null there the decoder?

Reply:
Good catch it should be default one BitmapFactoryDecoder

Reply:
@giannis-rdx Yes I did look at the code.
Null is a default one for decoder, however If you look at the coil code there are 4 decoders, `SVGDecoder`, `GIFDecoder`, `ImageDecoderDecoder` and `BitmapFactoryDecoder`. 
`BitmapFactoryDecoder` is the base decoder if you look at the top of the file.

I tested also using null and standard images are also displayed properly. Same thing with base `BitmapFactoryDecoder` so that's why I feel like its more intuitive to use base one rather than null as its confusing for people when looking at this.

Reply:
thank you Raf!